packages/apollo-link-ws/README.md

```diff
@@ -0,0 +1,43 @@
+# WebSocket Link
+
+## Purpose
+An Apollo Link to allow sending a request over a web socket.
+
+## Installation
+
+`npm install apollo-link-ws --save`
+
+To use this package in a web browser or mobile app, you'll need a build system capable of loading NPM packages on the client.
+Some common choices include Browserify, Webpack, and Meteor +1.3.
+
+## Usage
+```js
+import WSLink from "apollo-link-ws";
+import { SubscriptionClient } from 'subscriptions-transport-ws';
+
+const GRAPHQL_ENDPOINT = 'ws://localhost:3000/graphql';
+
+const client = new SubscriptionClient(GRAPHQL_ENDPOINT, {
+  reconnect: true,
+});
+
+const link = new WSLink(client);

```

Comment:
AFAIK in the new world `client` is already a Link

Reply:
it has the same request api but isn't actually a link (no split, from, etc). Though since its terminating that shouldn't really matter. I think we should investigate making it a full link though